.husky/pre-commit

```diff
@@ -6,7 +6,7 @@ CHANGED_FILES=$(git --no-pager diff --name-only --cached --diff-filter=AM --rela
   '*.md' '*.html' '*.yml' '*.yaml' '*.json' '*.js' '*.jsx' '*.ts' '*.tsx' '*.css')
 
 if [ -n "$CHANGED_FILES" ]; then
-  pnpm dlx prettier --check $CHANGED_FILES
+  echo "$CHANGED_FILES" | xargs npx prettier --check
```

Comment:
That aint entirely right I would say. If you want to use the pretty installed via package.json (which makes sense and should've already been the case ... mea culpa) we should use pnpm exec or actually use the package.json script and pass some params to it.